website/docs/02-About/3-branding.mdx

```diff
@@ -12,33 +12,163 @@ This file hosts the organization's brand assets and guidelines for moja global.
-| Logo (White Background) | <img src="/img/moja-global-logo-white-background.png" width="100"/> |
+### Primary Logomark
 
-Against dark backgrounds, use the white versions of the logo. You can download the logos from the [images](https://github.com/moja-global/About_moja_global/blob/master/images) directory.
+Shown here is the primary Moja Global logomark.
 
-### Typography
+The primary logomark should be used carefully and in most cases in order to uphold clear and consistent communication.
 
-- **Primary Typeface**: Source Sans pro (Used on the [Website](https://moja.global/))
-- **Secondary Typeface**: Ropa Sans (Used on the Presentation)
+| Description  | Image |
+| ------------- |:-------------:|
+| Primary Colorway 1A is to be used in instances where a white/light colored background and/or a light colored image is present.      | <img src="/img/brand/primary-logomark-1a.png" alt="primary logomark 1a" width="150"/>     |
+| Primary Colorway 1B is to be used in instances where a black/dark colored background and/or a light colored image is present.      | <img src="/img/brand/primary-logomark-1b.png" alt="primary logomark 1b" width="150"/>     |
 
-Secondary Typeface is used only when the Primary Typeface is not available.
+#### Monochromatic Usage
 
-### Colours
+Shown here are the monochromatic versions of the primary logomark. They are to be only used in cases where color print is restricted.
 
-- **Main Color**: `#699b98`
-- **Accent Color 1**: `#808080`
+| Description  | Image |
+| ------------- |:-------------:|
+| Primary Colorway 1A - is to be used in instances where a white/light background is present.      | <img src="/img/brand/primary-logomark-mono-1a.png" alt="primary logomark mono 1a" width="150"/> |
+| Primary Colorway 1B is to be used in instances where a black/dark background is present.      | <img src="/img/brand/primary-logomark-mono-1b.png" alt="primary logomark mono 1b" width="150"/> |
 
-The Accent Color is to be rarely used. The Main Color should be used across Font, Banners, and Menus.
+### Secondary Logomark
 
-### Imagery
+Shown here is the secondary Moja Global logomark.
 
-The images should be high-quality, engaging photography that portrays images of landscapes, forests, plants, and communities, and individuals interacting with the landscape.
+The secondary logomark should be used carefully and in cases where the brand graphics are already present on the page.
 
-An example shown below is included in the [images](https://github.com/moja-global/About_moja_global/blob/master/images) directory. Please ensure that proprietary photographs are credited if the source is known.
+| Description  | Image |
+| ------------- |:-------------:|
+| Secondary Colorway 2A - is to be used in instances where a white/light colored background and/or a light colored image is present.      | <img src="/img/brand/secondary-logomark-2a.png" alt="secondary logomark 2a" width="150"/> |
```

Comment:
The alt-text should be descriptive of the image itself. Some of the alt-text you wrote is quite good and could even be incorporated into the text, but it needs to be reformulated into actual alt-text for the images. [Here's a nice guide about writing alt-text](https://webaim.org/techniques/alttext/).